Dockerfile

```diff
@@ -1,25 +1,39 @@
-FROM golang:1.13-alpine as builder
+FROM golang:1.13-alpine as base
+ENV GOOS=linux CGO_ENABLED=0 GOARCH=amd64
 RUN apk --no-cache add git
-WORKDIR /
-COPY . /
-RUN GOOS=linux CGO_ENABLED=0 GOARCH=amd64 go build -o conftest -ldflags="-w -s" cmd/main.go
+WORKDIR /app
 
-FROM golang:1.13-alpine as test
-COPY --from=builder /conftest /
-RUN go test /conftest/...
+COPY go.mod .
+COPY go.sum .
```

Comment:
nitpick: `COPY go.mod go.sum .`

Reply:
I'm not super opposed to the proposed line change, but I tend to prefer doing it line by line (in this case) for readability.